rakefile.netstd.rb

```diff
@@ -106,9 +106,17 @@
 end
 
 desc "Restore NuGet packages"
-exec :restore do |cmd|
-  cmd.command = nuget_command
-  cmd.parameters "restore #{solution} -PackagesDirectory #{packages}"
+task :restore do
+  restore_cmd = Exec.new
+  restore_cmd.command = nuget_command
+  restore_cmd.parameters "restore #{solution} -PackagesDirectory #{packages}"
+  restore_cmd.execute
+
+  # performing restore on the solution doesn't restore

```

Comment:
Yep, if it needs to be done then it needs to be done. Definitely bizarre behaviour though.